mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/presenter/DebitCardPresenter.java

```diff
@@ -27,8 +27,12 @@ public void attachView(BaseView baseView) {
     @Override
     public void verifyDebitCard(String s, String s1, String s2) {
         String otp = "0000";
-        mDebitCardView.verifyDebitCardSuccess(otp);
-//        String message = "error";
-//        mDebitCardView.verifyDebitCardError(message);
+        if((s.length() < 12) || (s.length() > 19)) {
```

Comment:
Here you have a check style violation that is why your build is failing.  Enter a space between `if` and `(`

Reply:
Oh, okay
I'll correct it.
Thank You